MetaMask/metamask-extension
# UX: Show Total Account Balances

## **Description**

Shows total account balances in the Accounts Menu and home screen overview.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Open the account list -- see account totals
2. See the same total on the home overview of each account

## **Screenshots/Recordings**

### **Before**

<img width="316" alt="SCR-20231206-kihi" src="https://github.com/MetaMask/metamask-extension/assets/46655/e4dd94bc-8392-40b9-8a5a-0cd03456790a">


### **After**

<img width="391" alt="SCR-20231206-kijz" src="https://github.com/MetaMask/metamask-extension/assets/46655/61c485bc-ff49-407e-9157-09cc50bd5ead">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
